logger/logger.go

```diff
@@ -73,132 +64,146 @@ func (l LogLevel) String() string {
 //
 /////////////////////////////
 
-// Logger provides customization of log messages. We can change globals, default log level, formatting, and output destination.
+// Logger is the default implementation of KayveeLogger.
+// It provides customization of globals, default log level, formatting, and output destination.
 type Logger struct {
 	globals   map[string]interface{}
 	logLvl    LogLevel
-	formatter Formatter
-	logWriter *log.Logger
+	fLogger   formatLogger
 	logRouter router.Router
 }
 
 // SetConfig allows configuration changes in one go
```

Comment:
nit: looks like this comment should be alongside the `SetConfig` method or removed